lib/mini_profiler/profiler.rb

```diff
@@ -336,7 +336,7 @@ def call(env)
         @storage.save(page_struct)
 
         # inject headers, script
-        if headers['Content-Type'] && status == 200
+        if status >= 200 && status < 300
```

Comment:
I removed `headers['Content-Type']` from the condition to set X-MiniProfiler-Ids when status is 204 (No Content). I believe this is not a problem since X-MiniProfiler-Ids was set regardless of `headers['Content-Type']` until c101e7463d6ebf6d3b0078e81e9a7b7ddd17fa18. `headers['Content-Type']` is still used to decide whether to inject js to body in inject_profiler, just like c101e7463d6ebf6d3b0078e81e9a7b7ddd17fa18 did.